### ISSUE TITLE
Question / Request

### ISSUE THREAD

**Issue by thekalbb** [Frvetz/FS22_SlipDestroysFields#10]
Hello, love the mod!

Would it be possible for you to add a slider to adjust at what level the slippage starts - that way you can make your map more difficult, or less.

**Comment by Frvetz** [comment]
Hello, thank you!!

That's a brilliant idea, I'm just not sure if the FS supports such a function. Do you know another mod that uses such a slider? 

Kind regards
Gerrit

**Comment by thekalbb** [comment]
In updateWheelFriction theres something called wheel.tireGroundFrictionCoeff - maybe that could be it?

https://gdn.giants-software.com/documentation_scripting_fs22.php?version=script&category=48&class=580#setWheelTireFrictionDirty8864

Theres also something called Wheelmass - which might be a global, across all tiretypes - lowering that might "nerf" all tire types.. But might come with unintended drawbacks..

**Comment by Frvetz** [comment]
Yeah, changing at which level the slippage starts is not really the problem here. I have already included a setting where you can change the strength... Only the slider is the problem 🤔

**Comment by thekalbb** [comment]
oh, i thourght the strenght ment how much "AOE" the ground would be set to cultivated - in that case
theres no need for a slider then, you allready have low, medium, high - thats a slider for enums xD

**Comment by Frvetz** [comment]
Alright :D Unfortunately, the FS is not 100% bug-free as far as the wheels are concerned and it is very difficult to find a "correct" value. It may also be that the other settings do not have the effect you would like, if that is the case you can simply write to me again and I'll try to adapt it a bit :)

If you want, you can also take a look at the file "registerSlipDestroysFields.lua". There you will find a tutorial on how to change the values yourself.
